plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -1287,7 +1291,8 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
                 :: BS.ByteString -> BS.ByteString -> BS.ByteString -> BuiltinResult Bool
             verifyEd25519SignatureDenotation =
                 case semvar of
-                  DefaultFunSemanticsVariant1 -> verifyEd25519Signature_V1
+                  DefaultFunSemanticsVariant0 -> verifyEd25519Signature_V1
+                  DefaultFunSemanticsVariant1 -> verifyEd25519Signature_V2
```

Comment:
This mapping is potentially easy to get wrong. Is there any test for this?

Reply:
We're just replacing one implementation with another for maintenance reasons, so doing this matching here is only for the purpose of being paranoid about not breaking anything in the past in case of some unknown unknown. There's nothing additional to test therefore beyond what we already test, which is how we use `ed25519_Variant0Prop` to test all variants of this builtin.

Note that `consByteString` is untestable in the same way, but for a different reason: `Variant0` and `Variant1` versions of that builtin are completely identical (unlike with `verifyEd25519Signature`, whose variants behave identically but come from different libraries, i.e. are nominally different), so again there's nothing to test.

So for both the builtins we only make sure that whatever their variant is, it behaves as expected, we cannot test that `Variant0` behaves differently from `Variant1`, because it doesn't.